packages/apps/plugins/plugin-deck/src/components/DeckLayout.tsx

```diff
@@ -304,10 +298,11 @@ export const DeckLayout = ({
               .map((id, index, main) => {
                 const node = resolveNodeFromSlug(graph, id);
                 const part = ['main', index, main.length] satisfies PartIdentifier;
+                const attendableAttrs = useAttendable(id);
                 return (
-                  <Deck.Plank key={id}>
+                  <Deck.Plank key={id} {...attendableAttrs}>
```

Comment:
Nice. I'm liking this data annotation approach.

Reply:
Yeah, I’m enjoying taking tabster’s lead on this, seems productive.